README-Docker.md

```diff
@@ -0,0 +1,68 @@
+# Docker
+
+
+## Create Docker image
+
+* Assuming you have [Docker](https://docs.docker.com/get-docker/) installed locally.
+* First create an image tagged `fish`
+  * The docker file will create an image with jekyll, java, node, sushi and the publisher jar file pre-installed. 
+
+```
+# build
+docker build -t fish .
+
+# validate
```

Comment:
I got confused this with "validating the IG," but it's more like a check-up for dependency installation, right?